package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@observablehq/lezer",
-  "version": "0.9.1",
+  "version": "0.0.0",
```

Comment:
Can we set this to `1.0.0` to get us started on semver? Starting with 0.x.x causes semver to do strange things, and effectively breaks `~` and `^` version ranges.